crates/llm/examples/inference.rs

```diff
@@ -15,8 +15,7 @@ fn main() {
 
     let now = std::time::Instant::now();
 
-    let architecture = ModelArchitecture::from_tag(model_type)
-        .unwrap_or_else(|| panic!("{model_type} is not a supported model"));
+    let architecture = model_type.parse().unwrap_or_else(|e| panic!("{e}"));
```

Comment:
Does the root-level `llm` example need to be changed to match this?

Reply:
No, that one just uses LLaMA statically for simplicity.